src/main/java/org/springframework/data/couchbase/repository/query/N1qlQueryCreator.java

```diff
@@ -60,18 +60,55 @@ private QueryCriteria from(final Part part, final CouchbasePersistentProperty pr
 														 final Iterator<Object> parameters) {
 
 		final Part.Type type = part.getType();
-
+/*
```

Comment:
these are still missing?

Reply:
Just two left now - NEAR and WITHIN.
WITHIN is straight-forward.  There doesn't appear to be a N1QL equivalent to NEAR.